qemu/tests/cfg/timedrift_adjust_time.cfg

```diff
@@ -7,7 +7,6 @@
     start_vm = no
     kill_vm = yes
     requries_root = yes
-    image_snapshot = yes
```

Comment:
Why remove this line?

Reply:
> Why remove this line?
Because there are vm.destroy() and vm.create() after setting new time zone in sync_timezone_win(). If don't remove snapshot, the new time zone wont take effect.